homeassistant/components/fivem/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow for FiveM integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from fivem import FiveM, FiveMServerOfflineError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_PORT = 30120
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME): cv.string,
```

Comment:
Please remove name from the form and use host as default title instead. The user can change title, device name and entity names afterwards if they like.